docs/src/apireference.md

```diff
@@ -216,6 +216,7 @@ Functions for getting and setting properties of sets.
 
 ```@docs
 output_dimension
+constant
```

Comment:
There is a way with Documenter to only make methods with functions appear here and methods with sets appear after
http://www.juliaopt.org/MathOptInterface.jl/dev/apireference/#MathOptInterface.dimension

Reply:
I didn't find anything for this in Documenter's doc, do you have the syntax for that? The only solution I see is a really ugly hack using `Filters`

Reply:
And didn't find anything like it in the rest of the docs

Reply:
Normally, you can do something like `constant(::MathOptInterface.AbstractSet)`

Reply:
See https://juliadocs.github.io/Documenter.jl/stable/man/guide/#Filtering-included-docstrings-1 and https://github.com/JuliaDocs/Documenter.jl/issues/569

Reply:
ok I managed to fix it. It's a very tricky process I wouldn't advise using it whenever it's avoidable